src/ensembl/src/content/app/entity-viewer/services/bookmarks/entity-viewer-bookmarks-service.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import storageService, {
+  StorageServiceInterface
+} from 'src/services/storage-service';
+import { PreviouslyViewedEntities } from 'src/content/app/entity-viewer/state/bookmarks/entityViewerBookmarksSlice';
+
+export enum StorageKeys {
+  BOOKMARKS = 'entityViewer.bookmarks',
+  PREVIOUSLY_VIEWED = 'entityViewer.previouslyViewedEntities'
+}
+
+export class EntityViewerBookmarksStorageService {
+  private storageService: StorageServiceInterface;
+
+  public constructor(storageService: StorageServiceInterface) {
+    this.storageService = storageService;
+  }
+
+  public getPreviouslyViewedEntities(): PreviouslyViewedEntities {
+    return this.storageService.get(StorageKeys.PREVIOUSLY_VIEWED) || {};
+  }
+
+  public updatePreviouslyViewedEntities(
+    activeGenomePreviouslyViewedEntities: PreviouslyViewedEntities
+  ) {
+    this.storageService.update(
+      StorageKeys.PREVIOUSLY_VIEWED,
+      activeGenomePreviouslyViewedEntities
+    );
+  }
+}
+
+export default new EntityViewerBookmarksStorageService(storageService);
```

Comment:
I think we can rename the file to `entity-viewer-bookmarks-storage-service.ts` to match wioth other storage service files.

Reply:
I would tentatively argue that the word "storage" when applied to this service is superfluous. If you look at the list of other services that store data:

![image](https://user-images.githubusercontent.com/6834224/109319056-5f7cf280-7846-11eb-8270-182a580890ea.png)

 and remove the word "storage" from them, you lose the sense of what those services are for. But if you remove the word "storage" from entityViewerBookmarksStorageService, it is still quite clear that this is a service responsible for handling bookmarks.

Reply:
I agree with @msmanoj as it is easy to identify what service it is for if it is specific to storage. 

But if we were to remove "storage" from here, shouldn't we change it in the above listed files too?